src/assets/stylesheets/style.css

```diff
@@ -8853,4 +8874,41 @@ body {
 .trans-margin {
   margin-bottom: 61px;
 }
+.importReport > div {
```

Comment:
Let's keep the css class names in small case separated by '-' as in the rest of the file.

Reply:
Oki... I have made the changes.